AcsEmulator/AcsEmulatorAPI/Models/CallAutomationModels.cs

```diff
@@ -1,4 +1,6 @@
 ﻿namespace AcsEmulatorAPI.Models
 {
-    public record CreateCallRequest(string CallbackUri);
+    public record CreateCallRequest(string CallbackUri, CommunicationIdentifierModel[] Targets);
```

Comment:
Let's make this match the request in the swagger including the optional properties: https://github.com/Azure/azure-rest-api-specs/blob/8d662e966f6de4cd85d3dd349b776336c687597d/specification/communication/data-plane/CallAutomation/stable/2023-10-15/communicationservicescallautomation.json#L1275